pkg/certificate/authorization/authorizations_test.go

```diff
@@ -43,24 +43,39 @@ var (
 	}
 )
 
-func TestCertSignerConfig_NewAuthDB(t *testing.T) {
+func TestNewDB(t *testing.T) {
 	ctx := testcontext.New(t)
 	defer ctx.Cleanup()
 
-	authDB, err := newTestAuthDB(ctx)
+	dbURL := "bolt://" + ctx.File("authorizations.db")
```

Comment:
Not sure if thats the right way to do that? iirc, there are elsewhere temp file being created.

Reply:
look ok to me. `ctx.Dir` and `ctx.File` are in temp location that will be removed with cleanup.

Reply:
I'm not sure what you mean.